Source/Core/VideoCommon/VideoConfig.cpp

```diff
@@ -41,7 +41,12 @@ VideoConfig::VideoConfig()
   backend_info.bSupportsBPTCTextures = false;
 
   bEnableValidationLayer = false;
+
+#if defined(ANDROID)
```

Comment:
The default is located here: https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/Core/Config/GraphicsSettings.cpp#L75

(this value will be overridden when Refresh() is called)